infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -242,12 +256,12 @@
      * a task in progress, this is a no-op.
      *
      * @param layout Layout to use to bootstrap the primary sequencer.
-     * @return Completable future which completes when the task completes successfully or with a
-     * failure.
+     * @return Future which completes when the task completes successfully or with a failure.
      */
-    public CompletableFuture<Boolean> triggerSequencerBootstrap(@NonNull Layout layout) {
+    public Future<Boolean> triggerSequencerBootstrap(@NonNull Layout layout) {
```

Comment:
I know that this method is not being used outside ManagementAgent, yet this is a breaking change of interface. Can we avoid it? Also if it is not needed to be public, now that the interface is changing maybe consider narrowing the visibility of triggerSequenceBootstrap method